infra/src/main/resources/db/migration/repeatable/R__04.06_track_number_in_layout_context.sql

```diff
@@ -31,7 +33,23 @@ select
   row.draft,
   row.change_user,
   row.change_time
-  from layout.track_number row
```

Comment:
Näihin voisi varmaan heittää jonkun kommentin että miksi nuo rivit haetaan samasta taulusta monta kertaa, ettei joku vain rupea "siistimään" tuota tietämättömyyttään. Olisi myös kiva jos keksisi jonkun tavan vähentää tämän logiikan duplikointia näiden 4 tiedoston välillä mutta se taitaa olla vähän hankala ongelma.

Reply:
Koska haettavien columnien nimet ovat samoja, niin _periaatteessa_ olisi mahdollista kirjoittaa funktio, joka luo sql-hakufragmentin näille haluttuun tauluun. Ei tosin missään mielessä ideaalinen ratkaisu, mutta ainoa joka mullekaan tulee mieleen... En tosin sanoisi että näitä on mitään järkeä lähteä muuttamaan siihen malliin, sillä vaikka tässä koodia duplikoituukin, niin sentään on selkeää että minkälainen haku näistä aina muodostuu.